builder/Makefile.release

```diff
@@ -186,3 +187,20 @@ endif
 clean:
 #
 	$(MAKE) clean
+
+TAGS := latest $(IMAGE_TAG) full full-$(IMAGE_TAG)
```

Comment:
Yep, you are right. I preferred this because I don't want to duplicate the `cosign sign` command for each build. But if you are okay with that, I can add the `cosign sign` command right after each build.